Making goals more concise + removing conformance language (#47)
cncf/cnf-wg: charter.md

The CNF WG operates under the aegis of CNCF. The focus of the CNF WG is to define the process around evaluating the cloud nativeness of networking applications, aka CNFs, running on Kubernetes. We collaborate with the [CNF test suite project](https://github.com/cncf/cnf-conformance/blob/master/README-testsuite.md) who works on the mechanics of the tests.

The goals for the group are:

> - To identify best practices for CNFs running on K8s which CNF Developers and operators alike may adopt

Reviewer: ```suggestion
- To identify best practices for CNFs running on Kubernetes which CNF Developers and operators alike may adopt
```

> - To identify best practices for CNFs running on Kubernetes which CNF Developers and operators alike may adopt

Reviewer: ```suggestion
- To identify cloud native best practices for CNFs running on Kubernetes which CNF Developers and operators alike may adopt
```

> - To determine which practices will allow networking applications to most efficiently utilize K8s capabilities and services

Reviewer 2: I think these goals are aligned with recent discussions. My only comment would be that this phrasing makes it sound like there are two different goals, each yielding a different set of best practices. Or maybe I am reading too much into this?

Author: @rannyh I think we will have 1 set of best practices though there may be multiple options that overlap. It was not intended that we have 2 sets. These goals could probably be combined or at least reworded.

The second could be taken as diving down into a set of best practices and choosing one that is more optimal for networking applications. The first may include multiple options that are both best practices. I expect we will have different options at times and we will provide details in the proposals about where they are most effective, caveats, etc.

Whatever the case, I was working with the existing goal wording with my suggested changes.  We could have another iteration in a new PR to adjust unless there are specific suggestions now.

Reviewer 2: I prefer we dedicate our efforts to working on the actual best practices, rather than beating the wording of the goals to death. We can keep it as is and re-visit this with a new PR in the future, if need be.

Author: @rannyh Agreed.  Please mark the PR as approved :)

> - To determine which practices will allow networking applications to most efficiently utilize K8s capabilities and services

Reviewer: ```suggestion
- To determine which practices will allow networking applications to most effectively utilize Kubernetes capabilities and services
```
